src/linker/utilities/slurm_utils.py

```diff
@@ -23,52 +23,6 @@ def get_slurm_drmaa() -> "drmaa":
     return drmaa
 
 
-def launch_slurm_job(
```

Comment:
To clarify, you're deleting this in the PR but a followon PR will actually do the slurm implementation?